elements/lisk-transactions/test/15_pom_transaction.spec.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+describe('Proof-of-misbehavior transaction', () => {
+	describe('validateAsset', () => {
+		it.todo(
+			'when header1.height is greater than or equal to header2.height but equal maxHeighPrevoted it should not return errors',
+		);
+
+		it.todo(
+			'when header1.height is greater than header2.maxHeightPreviouslyForged it should not return errors',
+		);
+
+		it.todo(
+			'when header1.maxHeightPrevoted is greater than header2.maxHeightPrevoted it should not return errors',
+		);
+
+		it.todo(
+			'when header1 is not contradicting header 2 it should return errors',
+		);
+
+		it.todo(
+			'when block header 1 or 2 are not properly signed it should return errors',
+		);
+	});
+
+	describe('applyAsset', () => {
+		it.todo('should add reward to balance of the sender');
```

Comment:
I'm not sure where this goes:
"compute reward, which is min(regular block reward at height h, misbehaving delegate balance - MIN_BALANCE)"

Reply:
yes it will be computed and checked in the `expect` when test is written.